test/runtests.cmd

```diff
@@ -323,15 +323,10 @@ goto :main
     set _DIRS=-all
   )
 
-  set _BuildArchMapped=%_BuildArch%
   set _BuildTypeMapped=%_BuildType%
 
-  :: Map new build arch and type names to old names until rl test tags are
-  :: updated to the new names
-  if "%_BuildArchMapped%" == "x64" set _BuildArchMapped=amd64

```

Comment:
There's no need to keep the reverse mapping.  No one uses this script with the old names.

That said, our `unittest` tests will need to have their `exclude_*` tags converted to be compatible with this change to `runtests.cmd`, which would then also require `unittest\RunAllRLTests.cmd` to be updated to do the reverse mapping so that snap still works.

I believe the internal `ProjectionTests` suite has a completely separate script for calling rl, and so should not be affected by this change. So fortunately merging this PR shouldn't require any work in the `ProjectionTests` suite.